### ISSUE TITLE
Create deviaton schedule page

### ISSUE THREAD

**Issue by Limpan** [teknik-eksjo/chronos#39]
Create the page that allows users to enter their deviation schedule. Depends on #37.

**Comment by Greenheart** [comment]
I've started on the front-end for this feature. I guess we can make another issue for the back-end further on.

---

## Progress
- [x] Create the route
- [x] Update `create-slider.js` to work for both `/deviations` and `/base-schedule` - Do not initialize all sliders at once when the script is run on `/deviations`, do it **when new deviation-components are added**
- [X] Add a button whose click-event add new deviations
- [ ] Bug: deviations get values from last edited slider, not the newDeviationTemplate with default options. Probably some issue with the cloning of objects. Maybe re-create based on dynamic HTML-elements?
- [ ] Add a button that remove deviations. (red bg, innerText='X'). Prompt user to confirm deletion
- [ ] Integrate some kind of **datepicker component**. Display it to the left of the slider instead of the weekday-label
- [ ] Modify the `change`-event-handler to send the `date` from the datepicker instead of the `index`
- [ ] Fix styling of the `.add-deviation`-button --> When clicked, a weird dark green color appears and almost instantly disappears.
- [ ] Add a submit-button that, if the submit-request is validated by the server, allow the user to continue to the next step in the process **(the summary)**

**Comment by shhmon** [comment]
I will take over this issue for now. As soon as I get the sliders working and they stop getting initiated with data from the last modified slider I will continue reporting progress on the feature.
